internal/net/ssl/ssl.go

```diff
@@ -341,7 +342,7 @@ func AddOrUpdateDHParam(name string, dh []byte) (string, error) {
 func GetFakeSSLCert() *ingress.SSLCert {
 	cert, key := getFakeHostSSLCert("ingress.local")
 
-	sslCert, err := CreateSSLCert(cert, key)
+	sslCert, err := CreateSSLCert(cert, key, "")
```

Comment:
So we can set that default UID here, that way we know for sure it is for fakesslcert

Reply:
fixed